x-pack/plugin/sql/qa/src/main/resources/agg.csv-spec

```diff
@@ -309,4 +309,41 @@ SELECT HISTOGRAM(emp_no % 100, 10) AS h, COUNT(*) as c FROM test_emp GROUP BY h
 20             |10             
 10             |10             
 0              |10   
-;
\ No newline at end of file
+;
+
+countAll
+schema::all_names:l|c:l
+SELECT COUNT(ALL first_name) all_names, COUNT(*) c FROM test_emp;
+
+   all_names   |       c
+---------------+---------------
+90             |100
+;
+
+countAllCountTypesWithHaving
+schema::ln:l|dln:l|fn:l|dfn:l|ccc:l
+SELECT COUNT(last_name) ln, COUNT(distinct last_name) dln, COUNT(first_name) fn, COUNT(distinct first_name) dfn, COUNT(*) ccc FROM test_emp GROUP BY gender HAVING dln>5 AND ln>32 AND dfn>1 AND fn>1 AND ccc>5;
+
+      ln       |     dln     |      fn       |     dfn    |      ccc
+---------------+-------------+---------------+------------+-------------
+33             |32           |32             |32          |33
+57             |54           |48             |48          |57
+;
+
+aggCountEqualityFalse
+schema::bool:b|ln:l|dln:l
+SELECT FALSE=(COUNT(last_name)=COUNT(DISTINCT last_name)) AS bool, COUNT(last_name) ln, COUNT(DISTINCT last_name) dln FROM test_emp;
```

Comment:
I would remove the comparison to FALSE and name the column `equal` or `areEqual` instead of `bool`.